.github/workflows/pull_requests.yml

```diff
@@ -0,0 +1,31 @@
+name: Pull Request
+
+on:
+  pull_request:
+    branches: [ main ]
+
+## We are cancelling previously triggered workflow runs
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+ 
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        swift:
+        - image: swift5.8.1-jammy
+        - image: swift5.9.2-jammy
```

Comment:
Can we use less specific tags so we automatically take updates?

Reply:
Yeah I did that intentionally but you are right we should just pick up the latest patch updates